scene/gui/text_edit.cpp

```diff
@@ -729,11 +729,6 @@ void TextEdit::_notification(int p_what) {
 					VisualServer::get_singleton()->canvas_item_add_rect(ci, Rect2(xmargin_beg, ofs_y, xmargin_end - xmargin_beg, get_row_height()), cache.mark_color);
 				}
 
-				if (text.is_breakpoint(line)) {
-
-					VisualServer::get_singleton()->canvas_item_add_rect(ci, Rect2(xmargin_beg, ofs_y, xmargin_end - xmargin_beg, get_row_height()), cache.breakpoint_color);
-				}
-
```

Comment:
Can't really remove this, as if the user has `show_breakpoint_gutter` to false they will have no visual clue to where breakpoints are.

Reply:
ok, then will change that so it does not look so intrusive.